frontend/src/app/degreeReq/degreeReq.component.html

```diff
@@ -24,7 +23,7 @@ <h3>To complete the degree, you would need to take the followings: </h3>
                 <p>
                     {{req.quantity}} of the following:
                     <span *ngFor="let course of req.course;let index=index;let last = last">
-                        <span *ngIf="index < 10">{{course}}<span *ngIf="!last">, </span> <span *ngIf="!last && index == 9"> and etc </span></span>
+                        <span *ngIf="index < 9" (click)="getPrereqGraph(course)">{{course}}<span *ngIf="!last">, </span> <span *ngIf="!last && index == 8"> and others </span></span>
```

Comment:
Can we make this be a button instead of a span? Or maybe we can just underline it in blue.
Otherwise, there is no indication that it is clickable.